### PR TITLE
Update 01-intro.adoc

### PR DESCRIPTION
**Pull request by ptc**
changed some wording

### CODE REVIEW

File: docs/src/docs/01-intro.adoc [hunk 1]
```diff
@@ -73,10 +73,10 @@ The Opt-In Layer (OIL) is our solution for these requirements consisting of thre
 
 Opt-In Information is stored in a first party cookie in the user's browser and communicated in a general way to the host site and Tealium. For Power Opt-In there is also a 3rd party cookie stored on a second domain to share information between sites.
 
-=== Why is there no native app support?
+=== Why there is no native app support?
```
Comment (phogel) [inline docs/src/docs/01-intro.adoc]: This is wrong and the original was correct.

File: docs/src/docs/01-intro.adoc [hunk 2]
```diff
@@ -73,10 +73,10 @@ The Opt-In Layer (OIL) is our solution for these requirements consisting of thre
...
 
 Native mobile apps are very customized concerning technology stack and implementation of tracking and advertising. Therefore, the OIL project won't offer a generic solution for this purpose.
-Each unit has to review their app how tracking and advertising is implemented and find a solution to get the consent from the user by a dialog.
-Consents both on desktop or mobile browsers given by the user are not able to be used in the mobile app because:
+Each unit has to review their app regarding the individual implementation of tracking and advertising and find a solution to get the user's consent using a dialog.
+Consents both on desktop or mobile browsers previously given by the user can't be used in the mobile app for two technical reasons:
```
Comment (phogel) [inline docs/src/docs/01-intro.adoc]: I'd remove the previously, the rest is OK

File: docs/src/docs/01-intro.adoc [hunk 3]
```diff
@@ -73,10 +73,10 @@ The Opt-In Layer (OIL) is our solution for these requirements consisting of thre
 
 Opt-In Information is stored in a first party cookie in the user's browser and communicated in a general way to the host site and Tealium. For Power Opt-In there is also a 3rd party cookie stored on a second domain to share information between sites.
 
-=== Why is there no native app support?
+=== Why there is no native app support?
 
 Native mobile apps are very customized concerning technology stack and implementation of tracking and advertising. Therefore, the OIL project won't offer a generic solution for this purpose.
-Each unit has to review their app how tracking and advertising is implemented and find a solution to get the consent from the user by a dialog.
-Consents both on desktop or mobile browsers given by the user are not able to be used in the mobile app because:
+Each unit has to review their app regarding the individual implementation of tracking and advertising and find a solution to get the user's consent using a dialog.
```
Comment (phogel) [inline docs/src/docs/01-intro.adoc]: Even with this correction I still don't really get the point of this sentence

Comment (ltparis2018) [reply docs/src/docs/01-intro.adoc]: I think this sentence is useless. "Check and find a solution by yourself" is not a useful advice. We should remove it.

**Pull request merged**